### ISSUE TITLE
Support reordering of when container in choice container

### ISSUE THREAD

**Issue by apupier** [KaotoIO/kaoto#1484]
### Please describe the feature that you want to propose

the order has a meaning and an importance for when inside choice. Allowing to reorder would be useful

**Comment by lhein** [comment]
I think we do already have an issue for that open.

**Comment by lhein** [comment]
relates to https://github.com/KaotoIO/kaoto/issues/341

**Comment by igarashitm** [comment]
doTry (doCatch) also have a same issue

**Comment by apupier** [comment]
relates to #338

**Comment by shivamG640** [comment]
Are we talking about adding "Move Left" or "Move Right" options (or "Move Up"/"Move Down" in a horizontal layout) to the When container's context menu here?

**Comment by lordrip** [comment]
@shivamG640 we're talking about reordering visually and reflecting it in the route. For example, you have 3 `when` steps, you could reorder them, and that should be reflected on the route as well.

**Comment by shivamG640** [comment]
> [@shivamG640](https://github.com/shivamG640) we're talking about reordering visually and reflecting it in the route. For example, you have 3 `when` steps, you could reorder them, and that should be reflected on the route as well.

Right, if we move the when container, the route choice node definition will also need to be realigned accordingly.
I just wanted to confirm whether we allow reordering through move up/down or left/right options in the context menu, or is there a different approach we're considering for this?

**Comment by lordrip** [comment]
I'm sorry, I misunderstood your question. Ideally, we should be able to do this by drag and drop, but we can explore other ideas.
